Bifrost.h

```diff
@@ -89,8 +91,24 @@ Bifrost::Bifrost(const std::string& img) : fInside(false)
     const std::string mount = fDir+":/bifrost";
 
     std::vector<const char*> args = {"singularity", "run", "--cleanenv",
-                                     "-B", mount.c_str(),
-                                     img.c_str()};
+                                     "-B", mount.c_str()};
+
+    std::string data_dir;
```

Comment:
No need for `data_dir` variable. You can just do `args.push_back((jf_data+":/jf_data").c_str());`

Reply:
I tried this first, but when I did, for some reason only an empty string would be pushed back into the vector. Maybe some sort of scope issue?

Reply:
ooh, right. Yes, the `c_str()` is pointing at the contents of the string object, which in my version is a temporary that immediately goes away. So you need the code exactly like you have it. Maybe worth a comment on L96 that this string needs to be at this top level scope so that it is still alive when the exec happens.